tests/framework/installer/settings.go

```diff
@@ -41,8 +42,16 @@ func readManifest(provider, filename string) string {
 	return string(contents)
 }
 
-func readManifestFromGithub(rookVersion, provider, filename string) string {
-	url := fmt.Sprintf("https://raw.githubusercontent.com/rook/rook/%s/cluster/examples/kubernetes/%s/%s", rookVersion, provider, filename)
+func buildURL(rookVersion, filename string) string {
+	re := regexp.MustCompile(`(?m)^v1.6.[0-9]{1,2}$`)
```

Comment:
The upgrade test is still using 1.6.x, but in the future will be updated to use 1.7.x, so looks like 1.7 also needs to be supported.